code/modules/mob/logout.dm

```diff
@@ -8,6 +8,10 @@
 			if(VH.attached == src)
 				returnToPool(VH)
 
+	if(spell_channeling)
+		var/spell/thespell = on_uattack.handlers[spell_channeling]["o"]
```

Comment:
The handlers are stored in an associative list with 'o' denoting the object and 'p' denoting the procname